ts/Extensions/Boost/WGLRenderer.ts

```diff
@@ -859,6 +862,11 @@ class WGLRenderer {
                 isYInside = y >= yMin && y <= yMax;
             }
 
+            // Scatter points outside zoomed range were visible (#19701)
```

Comment:
Describe how the code actually works, not how it used to fail at some point:
```suggestion
            // Do not render points outside the zoomed range (#19701)
```